daffodil-core/src/main/scala/org/apache/daffodil/runtime1/ChoiceTermRuntime1Mixin.scala

```diff
@@ -178,10 +178,13 @@ trait ChoiceTermRuntime1Mixin { self: ChoiceTermBase =>
 
   final lazy val modelGroupRuntimeData = choiceRuntimeData
 
+
+
+
   final lazy val choiceRuntimeData = {
     new ChoiceRuntimeData(
       position,
-      Delay(partialNextElementResolver),
+      Delay(partialNextElementResolver, 'ChoicePartialNextElementResolver),
```

Comment:
Does this not need a `val ...Delay` variable so that it can be forced to be evaluated?

Reply:
Ah, nevermind, I see now. The initialize function on these runtime objects is responsible for forcing delays, and that function is in a `requiredEvaluation...``. Makes sense.